### ISSUE TITLE
LongDurationStorage is not guaranteed to exist

### ISSUE THREAD

**Issue by cfe316** [GenXProject/GenX#116]
I'm trying out the `main` branch, SmallNewEngland/OneZone example. 
It's throwing an error on load because `setup["LongDurationStorage"]` does not exist.

This can probably be solved by calling `configure_settings` function in the `Run.jl` files

**Comment by dharik13** [comment]
LongDurationStorage is no longer read from the Setup file but from the generators_data.csv file see load_generators_data.jl

**Comment by cfe316** [comment]
The issue is that `generate_model.jl` references `setup["LongDurationStorage"]`

See 
```
162     # Model constraints, variables, expression related to energy storage modeling
163     if !isempty(inputs["STOR_ALL"])
164         EP = storage(EP, inputs, setup["Reserves"], setup["OperationWrapping"], setup["LongDurationStorage"], setup["EnergyShareRequirement"], setup["CapacityReserveMargin"], setup["StorageLosses"])
165     end
```

**Comment by dharik13** [comment]
Yes, I dont get why we were not passing setup entirely rather than passing specific keys of the dictionary. Need to be fixed. Feel free to fix it and push PR.

**Comment by cfe316** [comment]
Should I be basing the fix off of `main` or `Dev`? I'm not sure what the plan is to merge those two back together, and I worry about making the merge more difficult.

**Comment by cfe316** [comment]
This was resolved on `Dev` by #124 .
